File: GTTS/INL.TrainingService.Models/SaveTrainingEventInstructors_Result.cs
using System.Collections.Generic;
using INL.Services.Models;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventInstructors_Result : BaseResult, ISaveTrainingEventInstructors_Result
    {
        public List<GetTrainingEventInstructor_Item> Collection { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-instructors_result.ts


import { GetTrainingEventInstructor_Item } from './get-training-event-instructor_item';
import { ISaveTrainingEventInstructors_Result } from './isave-training-event-instructors_result';

export class SaveTrainingEventInstructors_Result implements ISaveTrainingEventInstructors_Result {
  
	public Collection?: GetTrainingEventInstructor_Item[];
  
	public ErrorMessages: string[] = null;
}






